src/mixpanel/index.js

```diff
@@ -2,40 +2,55 @@ import mixpanel from 'mixpanel-browser';
 
 const BROWSER_MIXPANEL_TOKEN = '9edede4b70de19f399736d5840872910';
 
-mixpanel.init(BROWSER_MIXPANEL_TOKEN);
-mixpanel.register({'timestamp': new Date().toString(), '$referrer': document.referrer});
-
-export const Mixpanel = {
-    get_distinct_id: () => {
-        return mixpanel.get_distinct_id();
-    },
-    identify: (id) => {
-        mixpanel.identify(id);
-    },
-    alias: (id) => {
-        mixpanel.alias(id);
-    },
-    track: (name, props) => {
-        mixpanel.track(name, props);
-    },
+let Mixpanel = {
+    get_distinct_id: () => {},
+    identify: () => {},
+    alias: () => {},
+    track: () => {},
     people: {
-        set: (props) => {
-            mixpanel.people.set(props);
-        },
-        set_once: (props)  => {
-            mixpanel.people.set_once(props)
-        }
+        set: () => {},
+        set_once: ()  => {}
     },
-    withTracking: async (name, fn, errorOpration = () => {}, finalOperation = () => {}) => {
-        try {
-            mixpanel.track(`${name} start`)
-            await fn();
-            mixpanel.track(`${name} finish`)
-        } catch (e) {
-            mixpanel.track(`${name} fail`, {error: e.message})
-            errorOpration(e)
-        } finally {
-            await finalOperation()
-        }
-  }
+    withTracking: () => {}
 };
+
+if (process.env.REACT_APP_NETWORK_ID || process.env.REACT_APP_IS_MAINNET) {
+    mixpanel.init(BROWSER_MIXPANEL_TOKEN);
```

Comment:
let's pass `BROWSER_MIXPANEL_TOKEN` in environment variable similar to `MIXPANEL_TOKEN`

```
if (process.env.MIXPANEL_TOKEN) {
    mixpanel.init(process.env.MIXPANEL_TOKEN, {'property_blacklist': ['$current_url']})
}
```

then if it's not set – tracking can be disabled, no need to tie it to specific networks

Reply:
need to add variable `BROWSER_MIXPANEL_TOKEN` to the testnet and betanet, too. But I forget the config file link. @vgrichina